src/pages/Whattodo.js

```diff
@@ -0,0 +1,33 @@
+import {useState, useEffect} from "react";
+import { FormattedMessage } from "react-intl";
+
+export default function Whattodo() {
+    const [Data, setData] = useState(null);
```

Comment:
```suggestion
    const [whatTodo, setWhatTodo] = useState(null);
```

nil: We should use variable name that's better communicate what that Data is for